src/simply-rets-shortcode.php

```diff
@@ -231,30 +231,38 @@ public static function attributeNameToParameter($name) {
      *   - params: an array of API search parameters
      *   - settings: a key/value of settings (non-search attributes)
      */
-    public static function parseShortcodeAttributes($atts, $setting_atts = array()) {
+    public static function parseShortcodeAttributes($atts, $setting_atts = array(), $explode_values = TRUE) {
         $attributes = array("params" => array(), "settings" => $setting_atts);
 
         if (!$atts) {
             return $attributes;
         }
 
         foreach ($atts as $param=>$value_) {
-            // Ensure "&" is not HTML encoded
-            // https://stackoverflow.com/a/20078112/3464723
-            $value = str_replace("&amp;", "&", $value_);
+
+            // 1. Escape values for use in HTML attributes
+            // 2. Ensure "&" is not HTML encoded (https://stackoverflow.com/a/20078112/3464723)
+            $value = str_replace("&amp;", "&", esc_attr($value_));
             $name = SrShortcodes::attributeNameToParameter($param);
 
             // Parse settings, don't add them to the API query
             if (array_key_exists($param, $setting_atts)) {
                 $attributes["settings"][$param] = $value;
             }
 
-            $values = explode(";", $value);
-            foreach($values as $idx=>$val) {
-                $values[$idx] = trim($val);
-            }
+            // By default, attributes with multiple values separated by
+            // a semicolon are split into an array. To byass this, set
+            // explode_values is set to false.
```

Comment:
I don't see where it's set to false. looks like TRUE

Reply:
The default is `TRUE` because that's how existing functions use it.

I needed to _not_ to this only in one new spot: https://github.com/SimplyRETS/simplyretswp/pull/277/files#diff-4ba915a2fc41cb5b9df1ecae7ae730c06ac08e38a2b702be265a0731bc6678e2R322-R326